_sass/sidebar.scss

```diff
@@ -8,7 +8,7 @@ nav {
 
 .nav-links {
   display: grid;
-  grid-template-columns: auto 100px 100px 100px 100px 100px 100px;
+  grid-template-columns: auto repeat(7, min-content);
```

Comment:
Great update here! It would be nice to get this to be dynamic in the future if we ever add more buttons, but that's future us 🙂

Reply:
Definitely, but any more anchors and there will likely not be any room left in the header 🙈! Already pretty tight as is.